.gitignore

```diff
@@ -20,3 +20,5 @@ testenv
 packages/@okta/i18n/src/json/
 packages/@okta/i18n/src/properties/*_in.*
 packages/@okta/i18n/dist/
+test-reports/
+package-lock.json
```

Comment:
* I notice that `package-lock.json` is not checked into the widget repo. Only `yarn.lock` is added. That's why I thought we don't want to check-in this file. Am I wrong?
* I'm using `yarn install` (because of existence of `yarn.lock` locally)
* Don't need `testcafe-reporter-xunit` dependency as it's an in-built reporter in testcafe